### PR TITLE
CMSP-293-object-cache-pro-ea-docs

### PR DESCRIPTION
**Pull request by jazzsequence**
## Summary

**[Object Cache Pro](https://docs.pantheon.io/guides/object-cache-pro)** - Adds a new guide for Object Cache Pro Early Access program

## Effect

The following changes are already committed:

* New `/object-cache-pro` guide/introduction page
* New `/object-cache-pro/installing-configuring` page for installation and configuration instructions

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] Markdown file headers are somewhat incomplete. I did my best to fill in what I could, but it should have some review/edits from the docs team.

### Dependencies and Timing

**Release**:
- [x] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by jazzsequence** [comment]
[Demo link](https://pr-8264-documentation.appa.pantheon.site/guides/object-cache-pro/)

File: source/content/guides/object-cache-pro/02-installing-configuring.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+
+There are more [configuration options](https://objectcache.pro/docs/configuration-options) that can be added here as well. We recommend using the above settings which are optimized for Pantheon if you don't know specifically what each option does.
+
+#### 2. Activate the plugin and enable Redis in the plugin
+
+Before attempting to activate the plugin, make sure you `git push` your changes up to your repository.
+
+You can activate the Object Cache Pro plugin from the admin, locally with Lando or via Terminus. In the WordPress admin, navigate to the Plugins page and activate Object Cache Pro. Then go to the Object Cache Pro settings(?) page, ensure that you are in an environment with file write permissions (either SFTP mode, if activating on your Pantheon Dev environment or in a local development environment), and click (?) the Enable (?) button. This will create the `object-cache.php` drop-in file. If you did this in your Pantheon Dev environment, be sure to commit the change in SFTP mode. If you did this locally, you'll want to commit the file to your repository.
```
Comment (jspellman814) [inline source/content/guides/object-cache-pro/02-installing-configuring.md:223]: Do the question marks in this paragraph mean that we have to confirm the behavior? I know the path for OCP settings is `wp-admin/options-general.php?page=objectcache` and I think the others are correct based on [OCP docs](https://objectcache.pro/docs/installation#activation).

File: source/content/guides/object-cache-pro/02-installing-configuring.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+### Using WordPress (Composer Managed)
+
+*For full configuration instructions, refer to the [official Object Cache Pro documentation](https://objectcache.pro/docs/composer-installation).*
+
+#### 1. Create the authentication token
+First, you must add your license token to Composer. You can do this with the following command.
+
+`comoser config --auth http-basic.objectcache.pro token <LICENSE-TOKEN>`
+
+Alternately, you can create an `auth.json` file (created in the above command) manually:
```
Comment (jspellman814) [inline source/content/guides/object-cache-pro/02-installing-configuring.md:81]:
```suggestion
Alternately, you can create an `auth.json` file (created by the above command) manually:
```

File: source/content/guides/object-cache-pro/02-installing-configuring.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+Once this is done, there are additional options that need to be added. [Skip down to the configuration options](#configuring-object-cache-pro) section to finish the setup process.
+
+### Using WordPress (Composer Managed)
+
+*For full configuration instructions, refer to the [official Object Cache Pro documentation](https://objectcache.pro/docs/composer-installation).*
+
+#### 1. Create the authentication token
+First, you must add your license token to Composer. You can do this with the following command.
+
+`comoser config --auth http-basic.objectcache.pro token <LICENSE-TOKEN>`
```
Comment (jspellman814) [inline source/content/guides/object-cache-pro/02-installing-configuring.md:79]:
```suggestion
`composer config --auth http-basic.objectcache.pro token <LICENSE-TOKEN>`
```

File: source/content/guides/object-cache-pro/02-installing-configuring.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+
+### On regular WordPress Sites
+
+#### 1. Install the Object Cache Pro plugin
+
+Refer to the email you received and the link to the zip file for Object Cache Pro. Download the file, unzip, and add to your WordPress install as you normally would. 
+
+*Note: This link will always point to the latest version of Object Cache Pro. Be sure to store it in a safe place so you never miss out on updates to Object Cache Pro.*
+
+Once you have Object Cache Pro in your codebase, there are some configuration options that you need to add to set it up. All of these options are stored in your `wp-config.php` in the root of your site repoistory.
```
Comment (jspellman814) [inline source/content/guides/object-cache-pro/02-installing-configuring.md:52]:
```suggestion
Once you have Object Cache Pro in your codebase, there are some configuration options that you need to add to set it up. All of these options are stored in your `wp-config.php` in the root of your site repository.
```

File: source/content/guides/object-cache-pro/01-introduction.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+permalink: docs/guides/object-cache-pro/
+editpath: object-cache-pro/01-introduction.md
+contenttype: [guide]
+innav: [true]
+categories: []
+cms: [wordpress]
+audience: [development]
+product: [--]
+integration: [--]
+reviewed: ""
```
Comment (jspellman814) [inline source/content/guides/object-cache-pro/01-introduction.md:19]: Leaving this empty displays an `Last Reviewed: Invalid date` message. We should add the go-live date or remove entirely. 🔢

**Review by jspellman814 — CHANGES_REQUESTED** [review]
Few small changes, but overall looks great. Do we want to note that folks should [delete](https://objectcache.pro/docs/installation#preparation) the `/wp-content/object-cache.php` drop-in if it already exists (e.g. for users of Redis Object Cache and WP Redis that want to try this out)?

File: source/content/guides/object-cache-pro/02-installing-configuring.md (hunk 1 above)
Comment (jazzsequence) [reply source/content/guides/object-cache-pro/02-installing-configuring.md:223]: Oh, good catch, yeah those were notes to myself for when I came back after testing! 🤣

File: source/content/guides/object-cache-pro/01-introduction.md (hunk 5 above)
Comment (jazzsequence) [reply source/content/guides/object-cache-pro/01-introduction.md:19]: I noted in the description that there's some stuff, particularly the file header, that should be filled out by the docs team.

**Comment by jazzsequence** [comment]
@jspellman814 

> Do we want to note that folks should [delete](https://objectcache.pro/docs/installation#preparation) the /wp-content/object-cache.php drop-in if it already exists (e.g. for users of Redis Object Cache and WP Redis that want to try this out)?

I don't think we need to. "Enabling" Object Cache Pro (either from the WP admin or WP CLI) creates/overwrites the file whether it exists already or not, so I don't know that we need to worry about whatever's already there (other than that you need to be in a writeable environment).

File: source/content/guides/object-cache-pro/01-introduction.md (hunk 5 above)
Comment (jspellman814) [reply source/content/guides/object-cache-pro/01-introduction.md:19]: Perfect, thanks!

File: source/content/guides/object-cache-pro/02-installing-configuring.md [hunk 6]
```diff
@@ -0,0 +1,316 @@
...
+			'enabled' => true,
+			'persist' => true,
+			'retention' => 3600, // 1 hour
+			'footnote' => true,
+			],
+		```
+
+1. Make sure you `git push` your changes up to your repository before you activate the plugin.
+
+1. Activate the Object Cache Pro plugin and enable Redis in the plugin from the WordPress Admin, locally with Lando, or via Terminus.
```
Comment (whitneymeredith) [inline source/content/guides/object-cache-pro/02-installing-configuring.md:104]: @jazzsequence This step states that Lando can be used but no steps are provided. Should example steps for Lando be added or should the reference be removed?

Comment (jazzsequence) [reply source/content/guides/object-cache-pro/02-installing-configuring.md:104]: Can remove the reference to Lando.

File: source/content/guides/object-cache-pro/02-installing-configuring.md [hunk 7]
```diff
@@ -0,0 +1,316 @@
...
+		'analytics' => [
+			'enabled' => true,
+			'persist' => true,
+			'retention' => 3600, // 1 hour
+			'footnote' => true,
+		],
+		```
+1. Make sure you `git push` your changes up to your repository before you activate the plugin.
+
+1. Activate the plugin and enable Redis in the plugin. You can activate the Object Cache Pro plugin from the WordPress Admin, locally with Lando, or via Terminus.
```
[the remark this answers is not in the document]
Comment (jazzsequence) [reply source/content/guides/object-cache-pro/02-installing-configuring.md:289]: Samesies.

[the remark this answers is not in the document]
Comment (jazzsequence) [reply source/content/guides/object-cache-pro/02-installing-configuring.md:289]:
```suggestion
1. Activate the plugin and enable Redis in the plugin. You can activate the Object Cache Pro plugin from the WordPress Admin, locally with WP-CLI, or via Terminus.
```

File: source/content/guides/object-cache-pro/02-installing-configuring.md (hunk 6 above)
Comment (jazzsequence) [reply source/content/guides/object-cache-pro/02-installing-configuring.md:104]:
```suggestion
1. Activate the Object Cache Pro plugin and enable Redis in the plugin from the WordPress Admin, locally with WP-CLI, or via Terminus.
```

Comment (jazzsequence) [reply source/content/guides/object-cache-pro/02-installing-configuring.md:104]:
```suggestion
1. Activate the Object Cache Pro plugin and enable Redis in the plugin from the WordPress Admin or via Terminus.
```

File: source/content/guides/object-cache-pro/02-installing-configuring.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (jazzsequence) [reply source/content/guides/object-cache-pro/02-installing-configuring.md:289]:
```suggestion
1. Activate the plugin and enable Redis in the plugin. You can activate the Object Cache Pro plugin from the WordPress Admin or via Terminus.
```

File: source/content/guides/object-cache-pro/02-installing-configuring.md (hunk 6 above)
Comment (whitneymeredith) [reply source/content/guides/object-cache-pro/02-installing-configuring.md]: Thanks @jazzsequence . I added WP-CLI.

**Comment by scottbuscemi** [comment]
Screenshot 2023-04-03 at 9 19 53 AM

I'm seeing some formatting errors with the code block

**Comment by jazzsequence** [comment]
@whitneymeredith do you have any idea why [this](https://github.com/pantheon-systems/documentation/pull/8264/files#diff-a7621e63f04748a1539d60159ba2cc07ff2e338982eece663b015c07ff88790eR148-R159) `json` code block works inside a numbered list:
Screen Shot 2023-04-05 at 11 00 56 AM

...but [this](https://github.com/pantheon-systems/documentation/pull/8264/files#diff-a7621e63f04748a1539d60159ba2cc07ff2e338982eece663b015c07ff88790eR167-R178) one doesn't behave the same?
Screen Shot 2023-04-05 at 11 01 05 AM

I think this is the same issue that @scottbuscemi is seeing on the preview page. If it's better to not define a language if we're doing code blocks inside lists, I can remove the backticks.

**Comment by jazzsequence** [comment]
@whitneymeredith this is good to merge if your changes did the trick.

**Pull request merged**